Content.Server/Cargo/Systems/CargoSystem.Bounty.cs

```diff
@@ -204,7 +211,7 @@ public bool IsBountyComplete(EntityUid container, EntityUid? station, out HashSe
             return false;
         }
 
-        station ??= _station.GetOwningStation(container);
+        station ??= component.AssociatedStationId;
```

Comment:
Is this null coalescing even needed anymore, seems like it just muddies control flow and hides what is happening.